### DISCUSSION TITLE
Neuere GPL-Codes: 5590 verfügbar 7.29+7.30

### DISCUSSION THREAD

**Discussion by maverick3005** [Freetz-NG/freetz-ng#774] [Allgemein]
Nach Anforderung bei AVM ist erst einmal etwas "Neueres final" verfügbar....
Leider lies sich AVM nicht überreden, den gewünschten kernel-Code 4.4.271 inkl. einer config vorab mit zu posten :-(

auf osp.avm.de sind jetzt neuere GPL-Codes verfügbar:

https://osp.avm.de/fritzbox/fritzbox-5590-fiber/

Für die 6890 kam leider nichts was halbwegs aktuell ist. Auch für die 7530ax lies sich AVM nicht überreden :-(

**Comment by fda77** [comment]
Nimmst du 
alten code
neuen code
und diff
Damit macht du einen patch
und packst ihn in make/kernel/patches/DAS/PASSENDE/
